server/get-info-from-googleapis.js

```diff
@@ -0,0 +1,92 @@
+const { google } = require('googleapis');
+const { spreadSheet } = require('./config');
+const _ = require('lodash');
+
+const TEAM_SHEET_ID = spreadSheet.id;
```

Comment:
@dmytropanontko , will it crash if any of the env-variables isn't provided?

Reply:
@epaminond  yes, it will be work but we will be have a error in console